server/database/queries/project/editProject.js

```diff
@@ -0,0 +1,27 @@
+const connection = require('../../config/connection');
+
+const editProject = (
+  name,
+  description,
+  imgUrl,
+  githubLink,
+  websiteLink,
+  projectType,
+  cohortId,
+  projectId,
+) =>
+  connection.query(
+    'UPDATE project SET name=$1, description=$2, img_url=$3, github_link=$4, website_link=$5, project_type=$6, cohort_id=$7 WHERE id = $8',
```

Comment:
@MohammedAlghazali In general it's best if PUT endpoints can handle requests that do not provide every argument, and can merge with existing data.
For example, it would be good if I could PUT `/projects/1` `{name: 'updated name'}` and then it updates only the name, while leaving the rest of the data, like description and img_url, set to their previous values.
This is a good design because then you can add new columns to the table and endpoint in the future, and if old clients are still doing PUT requests that don't have that new column, they won't nullify data in the mysql table.
In other words, what if you release an iOS app which is PUTting these data columns, then next month you add a new column, `project_owner`, and your web app can update the `project_owner` but people are still using the old version of the iOS app to edit projects, and that app does not know about `project_owner` and is still making PUT requests without `project_owner` - you want your endpoint to still work for those old apps.

I think you could accomplish this by passing the request body object to this function, instead of all of the destructured arguments, and then doing `UPDATE project SET ?`.  This link has more info:
https://stackoverflow.com/questions/59959755/update-data-with-nodejs/59960177#59960177